src/scss/components/_badge.scss

```diff
@@ -14,3 +14,12 @@
     height: calc-em(90);
   }
 }
+
+// colors
+.badge-purple {
+  @include badge-variant($sh-purple);
+}
+
+.badge-blue {
```

Comment:
why do you need 2 badges here?

Reply:
these class for badge background color, before i add purple and blue colors to the $theme-colors variables but i just realized we just only need for bg color so i remove additional colors from $theme-colors then create specific class for background.